Gopkg.toml

```diff
@@ -30,3 +30,7 @@ required = [
   # https://jira.percona.com/browse/PMM-1561
   name = "github.com/hashicorp/consul"
   version = "=0.8.5"
+
+[[constraint]]
+  name = "github.com/lib/pq"
+  version = "1.0.0"
```

Comment:
We keep only special constraints like about. That one is normal. Please remove it.